scala-conductr-client-lib/src/main/scala/com/typesafe/conductr/clientlib/scala/StatusCodes.scala

```diff
@@ -0,0 +1,11 @@
+package com.typesafe.conductr.clientlib.scala
+
+private[scala] object StatusCodes {
+  val Ok = 200
```

Comment:
These perhaps should be final, but I also think that they should be private to Scala. Both Akka and Play will expose their own constants.